src/test/kotlin/org/rust/cargo/util/CargoWithToolchainCommandCompletionProviderTest.kt

```diff
@@ -0,0 +1,54 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.cargo.util
+
+import com.intellij.codeInsight.completion.PlainPrefixMatcher
+import org.rust.cargo.RsWithToolchainTestBase
+import org.rust.cargo.project.model.cargoProjects
+
+class CargoWithToolchainCommandCompletionProviderTest : RsWithToolchainTestBase() {
+    fun `test target triple completion includes popular targets`() {
+        val testProject = buildProject {
+            toml("Cargo.toml", """
+                [package]
+                name = "hello"
+                version = "0.1.0"
+                authors = []
+            """)
+
+            dir("src") {
+                rust("main.rs", """
+                    fn main() {}
+                """)
+            }
+        }
+
+        val cargoProjects = testProject.psiFile("src/main.rs").project.cargoProjects
+        val cargoProject = cargoProjects.allProjects.first()
+        val provider = CargoCommandCompletionProvider(cargoProjects, cargoProject.workspace)
+        val (ctx, prefix) = provider.splitContextPrefix("build --target ")
+        val matcher = PlainPrefixMatcher(prefix)
+
+        val completions = provider.complete(ctx).filter { matcher.isStartMatch(it) }.map { it.lookupString }
+
+        assertContainsElements(
```

Comment:
As I can see, we use the same code in `org.rust.cargo.util.CargoCommandCompletionProviderTest#checkCompletion`.
Can we extract it into static method or event in some helper class to reuse and avoid code duplication?